### PR TITLE
Update from deprecated version of networkVersion

### PR DESCRIPTION
**Pull request by Espequair**
https://docs.metamask.io/guide/ethereum-provider.html#ethereum-networkversion-deprecated indicates `ethereum.networkVerstion` is now deprecated.
I'm proposing we change it to the link I put, since I can't find another place it's explicit.

### CODE REVIEW

**Comment by adilanchian** [comment]
Ya this is SICK - I feel like it may be worth pointing out here what they point out in the docs - 

`It's preferred to use the ChainID instead of the NetworkId` 

Maybe you can throw a little example of what this looks like and see if it works!!

Thoughts?

**Comment by adilanchian** [comment]
cc @farzaa @DavidBarrick 

This is a cool one!

**Comment by Espequair** [comment]
Sure, I'll get on it!

**Comment by DavidBarrick** [comment]
I like. Shouldn't the link point here thought? https://docs.metamask.io/guide/ethereum-provider.html#chain-ids

**Comment by Espequair** [comment]
That *should* be it. As to why I didn't just change the link to https://docs.metamask.io/guide/ethereum-provider.html#chain-ids, it's because it doesn't actually say how to fetch the chainID

File: NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md [hunk 1]
```diff
@@ -68,7 +68,23 @@ Hint, you'll need something in solidity called `require`. And, you'll like also
...
 Your website is **only** going to work on Rinkeby (since that's where your contract lives).
 
-Be sure to add a nice message somewhere telling users about this. You can also make it dynamic if you want and [detect their network](https://ethereum.stackexchange.com/questions/85194/how-to-check-the-current-metamask-network) and then say something like, "Hey — I see you're connected to mainnet but this only works on Rinkeby!". You can make this as fancy as you want!
+We're going to to add a nice message telling users about this! 
+
+For that, we make a RPC request to the blockchain to see the ID of the chain our wallet connects to. (Why a chain and not a network? [Good question!](https://ethereum.stackexchange.com/questions/37533/what-is-a-chainid-in-ethereum-how-is-it-different-than-networkid-and-how-is-it))
+
+We have already addressed requests to the blockchain. We used `ethereum.request` with the methods `eth_accounts` and `eth_requestAccounts`. Now we use `eth_chainID` to get the ID.
+
+```js
```
Comment (adilanchian) [inline NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md:77]: Can you change this to be `javascript` please :). Our App Markdown reader like that instead hehe

File: NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md [hunk 2]
```diff
@@ -68,7 +68,23 @@ Hint, you'll need something in solidity called `require`. And, you'll like also
...
+let chainID = await ethereum.request({ method: 'eth_chainId' });
+console.log("Connected to chain " + chainID);
+if (chainID !== "0x4" /*String, hex code of the chain ID of the Rinkebey test network */ ) {
+	alert("This website connects to a contract on the Rinkeby test network\nYour wallet is currently not connected to that network, so it won't work");
```
Comment (adilanchian) [inline NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md:81]: Can we just make this alert message say:

"You are not connected to the Rinkeby Test Network!"

Thoughts?

File: NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md [hunk 3]
```diff
@@ -68,7 +68,23 @@ Hint, you'll need something in solidity called `require`. And, you'll like also
...
+}
+```
+There, now the user will know if they're on the wrong network! 
+The request conforms to [EIP-695](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-695.md) so returns the hex value of the network as a string.
```
Comment (adilanchian) [inline NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md:85]: Small typo here:

`...so it returns the hex value of the network as a string.`

File: NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md [hunk 4]
```diff
@@ -68,7 +68,23 @@ Hint, you'll need something in solidity called `require`. And, you'll like also
...
+You can find other the IDs of the other networks on [here](https://docs.metamask.io/guide/ethereum-provider.html#chain-ids).
```
Comment (adilanchian) [inline NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md:86]: Small typo here:

`You can find IDs of other networks here`

**Review by adilanchian — CHANGES_REQUESTED** [review]
This is a great change! 

Left some comments for ya :)

File: NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md [hunk 5]
```diff
@@ -68,7 +68,23 @@ Hint, you'll need something in solidity called `require`. And, you'll like also
...
+We have already addressed requests to the blockchain. We used `ethereum.request` with the methods `eth_accounts` and `eth_requestAccounts`. Now we use `eth_chainID` to get the ID.
+
+```javascript
+let chainID = await ethereum.request({ method: 'eth_chainId' });
+console.log("Connected to chain " + chainID);
+if (chainID !== "0x4" /*String, hex code of the chain ID of the Rinkebey test network */ ) {
+	alert("You are not connected to the Rinkeby Test Network!");
+}
+```
+There, now the user will know if they're on the wrong network!
```
Comment (adilanchian) [inline NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md:84]: This should also be `their` and not `they're` btw :)

File: NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md [hunk 6]
```diff
@@ -68,7 +68,23 @@ Hint, you'll need something in solidity called `require`. And, you'll like also
...
+```javascript
+let chainID = await ethereum.request({ method: 'eth_chainId' });
+console.log("Connected to chain " + chainID);
+if (chainID !== "0x4" /*String, hex code of the chain ID of the Rinkebey test network */ ) {
+	alert("You are not connected to the Rinkeby Test Network!");
+}
+```
+There, now the user will know if they're on the wrong network! 
+The request conforms to [EIP-695](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-695.md) so it returns the hex value of the network as a string.
+You can find other the IDs of the other networks [here](https://docs.metamask.io/guide/ethereum-provider.html#chain-ids).
```
Comment (adilanchian) [inline NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md:86]: Can you change to `You can find IDs of other networks here!`

**Review by adilanchian — CHANGES_REQUESTED** [review]
Couple more small things here!

File: NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md (hunk 5 above)
Comment (Espequair) [reply NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md:84]: I'm actually pretty sure I'm right on this one. "They are on the wrong network"

Changed it to remove the contraction.

File: NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md [hunk 7]
```diff
@@ -68,7 +68,23 @@ Hint, you'll need something in solidity called `require`. And, you'll like also
...
 
-Be sure to add a nice message somewhere telling users about this. You can also make it dynamic if you want and [detect their network](https://ethereum.stackexchange.com/questions/85194/how-to-check-the-current-metamask-network) and then say something like, "Hey — I see you're connected to mainnet but this only works on Rinkeby!". You can make this as fancy as you want!
+We're going to to add a nice message telling users about this! 
+
+For that, we make a RPC request to the blockchain to see the ID of the chain our wallet connects to. (Why a chain and not a network? [Good question!](https://ethereum.stackexchange.com/questions/37533/what-is-a-chainid-in-ethereum-how-is-it-different-than-networkid-and-how-is-it))
+
+We have already addressed requests to the blockchain. We used `ethereum.request` with the methods `eth_accounts` and `eth_requestAccounts`. Now we use `eth_chainID` to get the ID.
+
+```javascript
+let chainID = await ethereum.request({ method: 'eth_chainId' });
```
[the remark this answers is not in the document]
Comment (Espequair) [reply NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md]: `chain_Id` as it turns out. I changed every instance to make sure it was consitent.

File: NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md (hunk 6 above)
Comment (Espequair) [reply NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md]: I swear I did my own proofreading pass first :(

File: NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md [hunk 8]
```diff
@@ -68,7 +68,23 @@ Hint, you'll need something in solidity called `require`. And, you'll like also
...
+We're going to to add a nice message telling users about this! 
+
+For that, we make a RPC request to the blockchain to see the ID of the chain our wallet connects to. (Why a chain and not a network? [Good question!](https://ethereum.stackexchange.com/questions/37533/what-is-a-chainid-in-ethereum-how-is-it-different-than-networkid-and-how-is-it))
+
+We have already addressed requests to the blockchain. We used `ethereum.request` with the methods `eth_accounts` and `eth_requestAccounts`. Now we use `eth_chainId` to get the ID.
+
+```javascript
+let chainId = await ethereum.request({ method: 'eth_chainId' });
+console.log("Connected to chain " + chainId);
+if (chainId !== "0x4" /*String, hex code of the chain ID of the Rinkebey test network */ ) {
```
Comment (adilanchian) [inline NFT_Collection/Section_5/Lesson_1_Finishing_Touches_Web_App.md:80]: Hey so sorry - one more change here

Can you just add this comment on the line above the if statement?

**Review by adilanchian — CHANGES_REQUESTED** [review]

**Comment by Espequair** [comment]
Change done. I'm writing this comment mostly to test if github will let me change the status

**Comment by adilanchian** [comment]
> Change done. I'm writing this comment mostly to test if github will let me change the status
> 
> **ETA:** No it won't, or at least there's another step I haven't discovered yet

Change the status? Hmmmm not quite sure what you mean here!

**Comment by adilanchian** [comment]
@Espequair thank you for working with my changes here!!

I went in and tweaked just a few things! This is GOOD TO GO.

**Comment by adilanchian** [comment]
@all-contributors please add @Espequair for doc

**Pull request merged**